lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -343,7 +343,7 @@
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
             <?php //@TODO @fenders look into adding a .-yelling class that all caps things ?>
-            <h3 class="inline-alt-text-color"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h3>
+            <h2 class="heading -emphasized"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h2>
```

Comment:
The [`.heading.-emphasized`](http://forge.dosomething.org/#heading) style modifier makes things all-caps as part of its visual appearance, so you can remove the `@TODO` on the line above and make the text sentence case again. ✏️